tests/form_params_test.go

```diff
@@ -57,99 +63,153 @@ func TestFormParametersMissingContentLength(t *testing.T) {
 	}
 }
 
-// Ensure passing a negative Content-Length or one that overflows results in a
+// Ensure passing a negative Content-Length or one that overflows integers results in a
 // 400 Bad Request
 func TestFormParametersBadContentLength(t *testing.T) {
-	reqBody := "veggie=potato"
-	postReq := []byte("POST / HTTP/1.1\r\n" + "Host: localhost:8080\r\n" + "Content-Type: application/x-www-form-urlencoded; charset=ASCII\r\n" + "Content-Length: -1" + reqBody + "\r\n" + "\r\n")
-	resp, err := requesttesting.MakeRequest(context.Background(), postReq, func(req *http.Request) {
-		t.Error("MakeRequest(): Expected handler not to be called.")
-	})
-	if err != nil {
-		t.Errorf("MakeRequest(): got err %v, want nil", err)
-	}
-	if !isBadReq(resp) {
-		t.Errorf("response: want %s, got %s", statusBadReq, string(resp))
-	}
 
```

Comment:
nit: Remove empty line in beginning of function

Reply:
resolved in [3786868](url)